tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3888,3 +3888,24 @@ def draw_bounding_boxes(images, boxes, name=None, colors=None):
     A `Tensor`. Has the same type as `images`.
   """
   return draw_bounding_boxes_v2(images, boxes, colors, name)
+
+
+@tf_export("image.generate_bounding_box_proposals")
+def generate_bounding_box_proposals(scores,
```

Comment:
@alextp could you help to review this change?

Reply:
I'm a little swamped; can you find someone else? Or I can look at it on friday.